lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -300,9 +305,18 @@ class RenderPopover extends RenderShiftedBox {
 
     context.canvas.drawPath(borderPath.shift(offset), _backgroundPaint);
 
-    if (child != null) {
-      context.paintChild(child!, offset + _contentOffset);
-    }
+    _clipPathLayer.layer = context.pushClipPath(
+      needsCompositing,
+      offset,
+      Offset.zero & child!.size,
```

Comment:
Why is `child!.size` the correct size? Shouldn't the height of this layer include the child height and the arrow height? I'm surprised that this works...

Reply:
When we want to extend the child over the arrow, the child's height already includes the arrow height.